app.py

```diff
@@ -1,8 +1,27 @@
-from flask import Flask
+from flask import Flask, request, render_template
+import requests
 
-app = Flask(__name__)
+app = Flask(__name__, template_folder='template')
 
-@app.route("/")
-def hello():
-    return "Hello, World!"
+@app.route('/')
+def index():
+    return render_template("index.html")    
+
+
+@app.route('/', methods=['POST'])
+def Onsubmit():
+    place = request.form['place']
+    # place_in_uppercase = place.upper()
```

Comment:
Don't keep the commented code with the working code

Reply:
okay, i don't do again this type of fault.Thank you